rocky/assets/js/renameReports.js

```diff
@@ -0,0 +1,35 @@
+import { onDomReady } from "../js/imports/utils.js";
+
+onDomReady(initConfirmButtons);
+
+export function initConfirmButtons(element) {
+  let root = element || document;
+  let confirm_buttons = root.querySelectorAll(".confirm-button");
+
+  confirm_buttons.forEach((button) => initClickHandlers(button));
+}
+
+function initClickHandlers(button) {
+  button.addEventListener("click", function (event) {
+    const target = event.target.closest("dialog");
+
+    editReportName(target);
```

Comment:
Should this show the dialog as well?

```suggestion
    editReportName(target);
    target.showModal();
```

Reply:
(Also, probably check if `target` isn't falsey, just in case?)

Reply:
It could, but the modal component takes care of the showing and hiding. This is only used to know to which context the fields within belong (we can query-select by class within the targets context, without having to worry that we might select other instances on the page).